src/main/java/seedu/address/model/person/Role.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.model.person;
+
+public class Role {
+    private final String role;
```

Comment:
We can use `public` since it is this class is immutable, and also to be consistent with other attribute.

Reply:
Agreed. Have made the change